pmm/pmm2-release-candidate.groovy

```diff
@@ -42,7 +42,6 @@ void runPMM2OVFBuild(String SUBMODULES_GIT_BRANCH, String RELEASE_CANDIDATE) {
 def pmm_submodules() {
     return [
         "pmm",
-        "qan-api2",
```

Comment:
What exactly happens when we remove this line? Don't we exclude the module from the build process?

Reply:
Yes, we do, but not the module, but the repo. It'll now be build from `percona/pmm` vs `percona/qan-api2`.